artman/cli/main.py

```diff
@@ -165,7 +165,13 @@ def parse_args(*args):
         '--image',
         default=ARTMAN_DOCKER_IMAGE,
         help=('[Optional] Specify docker image used by artman when running in '
-              'a Docker instance. Default to `%s`' % ARTMAN_DOCKER_IMAGE)),
+              'a Docker instance. Default to `%s`' % ARTMAN_DOCKER_IMAGE))
+    parser.add_argument(
+        '--generator_args',
```

Comment:
```suggestion
        '--generator-args',
```

(to match the other arguments in artman)